packages/shared/src/components/modals/OnboardingModal.tsx

```diff
@@ -70,8 +70,26 @@ function OnboardingModal({
     if (forceClose) {
       return onRequestClose(e);
     }
-
-    return setIsClosing(true);
+    const options: PromptOptions = {
```

Comment:
These can also go outside the component.

Reply:
Done